code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -31,6 +31,63 @@
 		else
 			update_inv_r_hand(FALSE)
 
+/mob/living/carbon/human/proc/do_kb_equip_slot_s_store()
+	do_kb_equip(SLOT_S_STORE)
+/mob/living/carbon/human/proc/do_kb_equip_slot_wear_suit()
+	do_kb_equip(SLOT_WEAR_SUIT)
+/mob/living/carbon/human/proc/do_kb_equip_slot_belt()
+	do_kb_equip(SLOT_BELT)
+/mob/living/carbon/human/proc/do_kb_equip_slot_back()
+	do_kb_equip(SLOT_BACK)
+/mob/living/carbon/human/proc/do_kb_equip_slot_boot()
+	do_kb_equip(SLOT_IN_BOOT)
+/mob/living/carbon/human/proc/do_kb_equip_slot_head()
+	do_kb_equip(SLOT_IN_HEAD)
+/mob/living/carbon/human/proc/do_kb_equip_slot_l_store()
+	do_kb_equip(SLOT_L_STORE)
+/mob/living/carbon/human/proc/do_kb_equip_slot_r_store()
+	do_kb_equip(SLOT_R_STORE)
+/mob/living/carbon/human/proc/do_kb_equip_slot_accessory()
+	do_kb_equip(SLOT_IN_ACCESSORY)
+/mob/living/carbon/human/proc/do_kb_equip_slot_beltholster()
+	do_kb_equip(SLOT_IN_HOLSTER)
+/mob/living/carbon/human/proc/do_kb_equip_slot_suitsholster()
+	do_kb_equip(SLOT_IN_S_HOLSTER)
+/mob/living/carbon/human/proc/do_kb_equip_slot_backholster()
+	do_kb_equip(SLOT_IN_B_HOLSTER)
+
+/mob/living/carbon/human/proc/do_kb_equip( KB )
+	SIGNAL_HANDLER_DOES_SLEEP
```

Comment:
This is not allowed

Reply:
literally copy from do_quick_equip so.. idk . but sure.